core/modules/simpletest/tests/simpletest.tests.info

```diff
@@ -958,6 +958,12 @@ description = Tests form API checkbox handling of various combinations of #defau
 group = Form API
 file = form.test
 
+[FormResizableTextareaTestCase]
+name = Form API resizable texarea
```

Comment:
Valid typo here:
```suggestion
name = Form API resizable textarea
```

Reply:
Oops... spellcheck can be useful. :wink: